frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -151,6 +159,25 @@ export function MainNavbarView({
             >
               {t`Home`}
             </PaddedSidebarLink>
+
+            {isEmbeddingHubFeatureEnabled && isAdmin && (
+              <PaddedSidebarLink
+                isSelected={nonEntityItem?.url === "/embedding-hub"}
+                icon={
+                  <Box
+                    w="10px"
+                    h="10px"
+                    bg="brand"
+                    mx="7px"
+                    className={CS.rounded}
+                  />
```

Comment:
Placeholder Icon. we do not have an icon for embedding hub yet.

Reply:
We got one in the design now, updated in https://github.com/metabase/metabase/pull/62517/commits/1f4bc2721415d95ad5c579aa3074fddcc11971b7.